jenkins-inbound-agent.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: jenkins-inbound-agent
+  version: 3077.vd69cf116da_6f-3
```

Comment:
Version is invalid.  https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-260003.3 outlines what can and cannot be used in a version string.

Reply:
That's a good question, we'll wait for @kaniini to get back to this. We'd want to make sure we produce a version that can be used downstream, e.g. in vuln matching.